File: py/palindrome-number.py
class Solution:
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if x >= 0 and x < 10:
            return True
        if x < 0 or x % 10 == 0:
          return False
        
        return str(x) == str(x)[::-1]

File: js/palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  if (x >= 0 && x < 10) return true
  if (x < 0 || x % 10 === 0) return false
  return x.toString() === x.toString().split('').reverse().join('')
};
